sdk/spring/azure-spring-boot-starter-active-directory-b2c/README.md

```diff
@@ -475,7 +475,7 @@ Please refer to [azure-spring-boot-sample-active-directory-b2c-resource-server].
 
 ## Troubleshooting
 ### Enable client logging
-Azure SDKs for Java offers a consistent logging story to help aid in troubleshooting application errors and expedite their resolution. The logs produced will capture the flow of an application before reaching the terminal state to help locate the root issue. View the [logging][logging] wiki for guidance about enabling logging.
+Azure SDKs for Java offers a consistent logging story to help aid in troubleshooting application errors and expedite their resolution. The logs produced will capture the flow of an application before reaching the terminal state to help locate the root issue. View the [logging][logging] doc for guidance about enabling logging.
 
 ### Enable Spring logging
 Spring allow all the supported logging systems to set logger levels set in the Spring Environment (for example, in application.properties) by using `logging.level.<logger-name>=<level>` where level is one of TRACE, DEBUG, INFO, WARN, ERROR, FATAL, or OFF. The root logger can be configured by using logging.level.root.
```

Comment:
Can we use property like `logging.level.<logger-name>=<level>` to configure the log level of Azure SDK?

Reply:
Yes, I have already experimented in my local area.